packages/teams/src/add-or-invite-users-to-teams.ts

```diff
@@ -0,0 +1,58 @@
+import {
+  ArcGISRequestError,
+  IAuthenticationManager,
+} from "@esri/arcgis-rest-request";
+import { addOrInviteUsersToTeam } from "./add-or-invite-users-to-team";
+import {
+  IAddOrInviteEmail,
+  IAddOrInviteToTeamResult,
+  IUserModalObject,
+} from "./types";
+
+export async function addOrInviteUsersToTeams(
+  groupIds: string[],
+  users: IUserModalObject[],
+  primaryRO: IAuthenticationManager,
+  canAutoAddUser: boolean = false,
```

Comment:
I don't even recall how this is determined - but is it accurate to have one value for say... 5 groups?

Reply:
If the current user has assignToGroups we allow them to directly add the user rather than invite them, so it has more to do with the current users privs than the groups in question.